master/master.cfg

```diff
@@ -889,6 +936,11 @@ def add_halide_cmake_test_steps(factory, builder_type):
         # env['HL_TARGET'] = halide_target
         env = extend_property('env', HL_JIT_TARGET=halide_target)
 
+        # wasm targets must ensure that the EMSDK (emcc, etc) are added to the
+        # active env.
+        if halide_target.startswith("wasm-"):
+            env = extend_with_emsdk_config(env)
```

Comment:
Then this would be:
```
env = merge_renderable(env, Property('emsdk_env'))
```

Reply:
Shamelessly plagiarized

Reply:
This might need to be `Property('emsdk_env', default={})`, but we'll see.